community/examples/intel/hpc-slurm-daos.yaml

```diff
@@ -121,13 +121,11 @@ deployment_groups:
       node_count_dynamic_max: 4
       machine_type: n2-standard-2
       enable_placement: false  # the default is: true
-      service_account:
-        email: null
-        scopes:
-        - "https://www.googleapis.com/auth/monitoring.write"
-        - "https://www.googleapis.com/auth/logging.write"
-        - "https://www.googleapis.com/auth/devstorage.read_only"
-        - "https://www.googleapis.com/auth/cloud-platform"
+      service_account_scopes:
+      - "https://www.googleapis.com/auth/monitoring.write"
```

Comment:
This is a weird set of scopes to be adding to the blueprint (`cloud-platform` is `*.*` and thus implies all other scope). Can you please create a task to reach out to Intel to update the blueprint? No action needed for this PR.